config/locales/en.yml

```diff
@@ -230,6 +230,10 @@ en:
   # should work here
   about_place: "About %{place}"
   about_places: About Places
+  about_taxon_ranges: About Taxon Ranges
+  about_taxon_ranges_desc: |
+    Taxon Ranges convey the global distribution of a taxon. Taxon ranges are
+    typically sourced from the IUCN RedList. <a href='%{url}'>Learn more</a>
```

Comment:
Translation keys that contain HTML should end in `_html`. That way you don't have to use `raw` when you use them in the templates.